org.audacityteam.Audacity.appdata.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component>
+  <id>org.audacityteam.Audacity</id>
+  <launchable type="desktop-id">org.audacityteam.Audacity.desktop</launchable>
+  <project_license>GPL-2.0 and CC-BY-3.0</project_license>
```

Comment:
At a glance source files seem to be `GPL-2.0-or-later`

Reply:
https://github.com/audacity/audacity/blob/master/LICENSE.txt